SCRIPTS.md

```diff
@@ -0,0 +1,10 @@
+# Scripts
+
+The following document includes information on the **NPM** scripts available via this project.
+Note that any script that includes `:ci` is intended to be used by our continuous integration platform only.
+
+* `storybook` - Starts the **Storybook** application and opens the browser.
+  * `storybook:build` - Builds the **Storybook** to the `./docs/storybook` directory.
+  * `storybook:ci` - Cleans the `./docs/storybook` directory and builds the **Storybook** to that directory.
+  * `storybook:clean` - Removes all directories and files from `./docs/storybook`.
+  * `storybook:watch` - Starts the **Storybook** application, opens the browser and watches the `./src` directory for changes.
```

Comment:
We should remove this one since it's not a valid command, see [discussion here](https://github.com/webex/components/pull/87/files#r385964192)